netsim/ansible/tasks/fortinet.fortios.fortios/ospf.yml

```diff
@@ -8,63 +8,69 @@
       auto_cost_ref_bandwidth: "{{ ospf.reference_bandwidth | default(omit) }}"
       router_id: "{{ ospf.router_id }}"
 
-- name: Configure OSPF on loopback
-  fortinet.fortios.fortios_router_ospf:
-    vdom: "{{ vdom }}"
-    router_ospf:
-      ospf_interface:
+- name: Add loopback0 to OSPF interfaces list
```

Comment:
I think you can get the same effect if you use **netlab_interfaces** (which includes the loopback). However, it looks like the loopback interface name needs to be changed in `netsim/devices/fortios.yml` (currently, it's Loopback).